mm2src/coins/utxo/utxo_block_header_storage/sql_block_header_storage.rs

```diff
@@ -303,22 +310,28 @@ impl BlockHeaderStorageOps for SqliteBlockHeadersStorage {
         })
     }
 
-    async fn remove_headers_up_to_height(&self, to_height: u64) -> Result<(), BlockHeaderStorageError> {
+    async fn remove_headers_from_storage(
+        &self,
+        from_height: u64,
+        to_height: u64,
+    ) -> Result<(), BlockHeaderStorageError> {
         let coin = self.ticker.clone();
```

Comment:
`self` is already cloned and ticker can be accesed from there, no need to clone this again

Reply:
done